assignment.rb

```diff
@@ -0,0 +1,103 @@
+# Accesses the csv file with user info
+require "csv"
+
+#Creates a User class with appropriate methods
+class User
+  attr_accessor :name, :pin
+  
+  def initialize(name, pin, balance)
+    @name = name
+    @pin = pin
+    @balance = balance
+  end
+
+  def name
+    @name
+  end
+
+  def pin
+    @pin
+  end
+
+  def balance
+    @balance.to_i
+  end
```

Comment:
these 3 methods already are created if you do
```ruby
attr_accessor :name, :pin, :balance
```
and then you can do the `.to_i` wherever you use `balance`